src/core/Akka.Cluster.Tests.MultiNode/MultiNodeLoggingConfig.cs

```diff
@@ -19,7 +19,7 @@ public static class MultiNodeLoggingConfig
 // ReSharper disable once InconsistentNaming
         private static readonly Config _loggingConfig =
             ConfigurationFactory.ParseString(@"
-                akka.loggers = [""Akka.Event.DefaultLogger""]");
```

Comment:
Removed the default logger as it wasn't needed, plus I think the FQN for this is actually wrong.